charts/descheduler/values.yaml

```
@@ -58,6 +58,9 @@ deschedulerPolicy:
 
 priorityClassName: system-cluster-critical
 
+#nodeSelector:
+#  foo: bar
```

I'd suggest we use the idiomatic way that _Helm_ sets these up for new charts.

```suggestion
nodeSelector: {}
  # foo: bar
```